cluster-autoscaler/utils/drain/drain_test.go

```diff
@@ -210,6 +210,150 @@ func TestDrain(t *testing.T) {
 		},
 	}
 
+	emptydirPodWithSafeToEvictLocalVolAnnotationSingleVal := &apiv1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:            "bar",
+			Namespace:       "default",
+			OwnerReferences: GenerateOwnerReferences(rc.Name, "ReplicationController", "core/v1", ""),
+			Annotations: map[string]string{
+				SafeToEvictLocalVolumeKey: "scratch",
+			},
+		},
+		Spec: apiv1.PodSpec{
+			NodeName: "node",
+			Volumes: []apiv1.Volume{
+				{
+					Name:         "scratch",
+					VolumeSource: apiv1.VolumeSource{EmptyDir: &apiv1.EmptyDirVolumeSource{Medium: ""}},
+				},
+			},
+		},
+	}
+
+	emptydirPodWithSafeToEvictLocalVolAnnotationSingleValEmpty := &apiv1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:            "bar",
+			Namespace:       "default",
+			OwnerReferences: GenerateOwnerReferences(rc.Name, "ReplicationController", "core/v1", ""),
+			Annotations: map[string]string{
+				SafeToEvictLocalVolumeKey: "",
+			},
+		},
+		Spec: apiv1.PodSpec{
+			NodeName: "node",
+			Volumes: []apiv1.Volume{
+				{
+					Name:         "scratch",
+					VolumeSource: apiv1.VolumeSource{EmptyDir: &apiv1.EmptyDirVolumeSource{Medium: ""}},
+				},
+			},
+		},
+	}
+
+	emptydirPodWithSafeToEvictLocalVolAnnotationSingleValNonMatching := &apiv1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:            "bar",
+			Namespace:       "default",
+			OwnerReferences: GenerateOwnerReferences(rc.Name, "ReplicationController", "core/v1", ""),
+			Annotations: map[string]string{
+				SafeToEvictLocalVolumeKey: "scratch-2",
+			},
+		},
+		Spec: apiv1.PodSpec{
+			NodeName: "node",
+			Volumes: []apiv1.Volume{
+				{
+					Name:         "scratch-1",
+					VolumeSource: apiv1.VolumeSource{EmptyDir: &apiv1.EmptyDirVolumeSource{Medium: ""}},
+				},
+			},
+		},
+	}
+
+	emptydirPodWithSafeToEvictLocalVolAnnotationMultiValAllMatching := &apiv1.Pod{
```

Comment:
Test names here e.g., `emptydirPodWithSafeToEvictLocalVolAnnotationMultiValAllMatching` feel a little too long. Any better suggestions are welcome :pray:

Reply:
`emptyDirSafeToEvictVolumeMultiValAllMatching` etc.? Seems to have more or less the same information.

Reply:
> `emptyDirSafeToEvictVolumeMultiValAllMatching` etc.? Seems to have more or less the same information.

I like this better. Thank you!

Reply:
Updated the names.